model/textparse/openmetricsparse.go

```diff
@@ -351,7 +461,15 @@ func (p *OpenMetricsParser) Next() (Entry, error) {
 			p.series = p.l.b[p.start:p.l.i]
 			t2 = p.nextToken()
 		}
-		return p.parseMetricSuffix(t2)
+
+		suffixEntry, err := p.parseMetricSuffix(t2)
```

Comment:
Nice, now it's much cleaner 💪🏽 

Do we need the same for `tBraceOpen` case?

Reply:
don't need it as it was actually causing test failures when I tried that